pages/api/hakkasan.ts

```diff
@@ -0,0 +1,54 @@
+import { NextApiResponse } from 'next';
+import withCache, { conditionallyUpdateCache } from '../../lib/withCache';
+import artists from '../../data/artists.json';
+import events from '../../data/events.json';
+import venues from '../../data/venues.json';
+import type { HakkasanRequest } from '../../lib/withCache';
+
+const handler = async (
+  req: HakkasanRequest,
+  res: NextApiResponse
+): Promise<void> => {
+  const {
+    method,
+    query: { ref },
+  } = req;
+  switch (method) {
+    case 'POST':
+      break;
+    case 'PUT':
+      break;
+    case 'PATCH':
+      break;
+    default: {
```

Comment:
We should explicitly use the `'GET'` case here instead of having it be `default`, especially considering the existing case statement don't include all http methods. For example, if the request method were to be `'HEAD'` it would resolve the request with an incorrect GET response.

Reply:
Oooh good point!!

Reply:
Looks good :) Resolving!